chain/network/src/routing/routing.rs

```diff
@@ -70,9 +70,8 @@ pub enum EdgeType {
 #[derive(BorshSerialize, BorshDeserialize, Clone, Debug, PartialEq, Eq)]
 #[cfg_attr(feature = "test_features", derive(Serialize, Deserialize))]
 pub struct Edge {
-    /// Since edges are not directed `peer0 < peer1` should hold.
-    pub peer0: PeerId,
```

Comment:
I'd suggest maybe removing `pub` from `key` and adding accessor methods.

Reply:
I made another PR for that. 

https://github.com/near/nearcore/pull/5223